<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>582</width>
    <height>438</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="NotOKBtn">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>380</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>NOT OK</string>
   </property>
  </widget>
  <widget class="QPushButton" name="CantTellBtn">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>380</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>CAN'T TELL</string>
   </property>
  </widget>
  <widget class="QLabel" name="flabel">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>170</y>
     <width>251</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Demonstrations Dictionary</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="DictionaryBox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>190</y>
     <width>531</width>
     <height>151</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="flabel_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>350</y>
     <width>221</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Operator Feedback</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="nameBox">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>30</y>
     <width>201</width>
     <height>21</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Enter User's name&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="flabel_3">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>30</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>User Name:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="skillcomboBox">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>70</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>red</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>yellow</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>brown</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>bumping</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>yellow_red</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="flabel_4">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>70</y>
     <width>151</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Skill/Behaviour:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="RunBtn">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>90</y>
     <width>113</width>
     <height>32</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Run</string>
   </property>
  </widget>
  <widget class="QPushButton" name="exitBtn">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>380</y>
     <width>113</width>
     <height>32</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
  </widget>
  <widget class="QPushButton" name="OKBtn">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>380</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
  <widget class="QLabel" name="flabel_5">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>100</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Iteration:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="ItCBox">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>100</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>1</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>2</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>3</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>4</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>5</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>6</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>7</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>8</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>9</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="flabel_6">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>130</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Frames:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="FCBox">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>130</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>1000</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>2000</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>3000</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>3500</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>4000</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>5000</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>6000</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>7000</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>8000</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>9000</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>10000</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>100000</string>
    </property>
   </item>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
